README.md

```diff
@@ -359,30 +361,33 @@ will kick off an asynchronous load of the specified media
 playlist. Once it has been retreived, it will become the active media
 playlist.
 
-#### hls.segmentXhrTime
+#### hls.systemBandwidth
 Type: `number`
 
-The number of milliseconds it took to download the last media segment.
-This value is updated after each segment download completes.
+`systemBandwidth` is a combination of two serial processes bit-rates. The first
+is the network bitrate provided by `bandwidth` and the second is the bitrate of
+the entire process after that - decryption, transmuxing, and appending - provided
+by `throughput`. This value is used by the default implementation of `selectPlaylist`
+to select an appropriate bitrate to play.
+
+Since the two process are serial, the overall system bandwidth is given by:
+`systemBandwidth = 1 / (1 / bandwidth + 1 / throughput)`
 
 #### hls.bandwidth
 Type: `number`
 
 The number of bits downloaded per second in the last segment download.
-This value is used by the default implementation of `selectPlaylist`
-to select an appropriate bitrate to play.
 
 Before the first video segment has been downloaded, it's hard to
-estimate bandwidth accurately. The HLS tech uses a heuristic based on
-the playlist download times to do this estimation by default. If you
+estimate bandwidth accurately. The HLS tech uses a starting value of 4194304 or 0.5 MB/s. If you
 have a more accurate source of bandwidth information, you can override
 this value as soon as the HLS tech has loaded to provide an initial
 bandwidth estimate.
 
-#### hls.bytesReceived
+#### hls.throughput
 Type: `number`
 
-The total number of content bytes downloaded by the HLS tech.
+The number of bits decrypted, transmuxed, and appended per second as a cumulative average.
```

Comment:
This reads a bit like it averages across all time, not just processing time. Might be worth it to add a clarification saying something like `(only averages across active processing time)`